src/store/traction/pacbio/requests/actions.js

```diff
@@ -1,11 +1,15 @@
-import handleResponse from '@/api/v1/ResponseHelper'
+import handleResponse from '@/api/v2/ResponseHelper'
```

Comment:
Just a thought, Better to include file extension which speeds up vite build process.